persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/ConversationDAOImpl.kt

```diff
@@ -445,8 +445,9 @@ class ConversationDAOImpl(
 
     override suspend fun revokeOneOnOneConversationsWithDeletedUser(userId: UserIDEntity) {
         conversationQueries.transaction {
-            val conversationId = memberQueries.selectConversationByMember(userId).executeAsOne().conversation
-            conversationQueries.updateConversationType(ConversationEntity.Type.GROUP, conversationId)
+            memberQueries.selectConversationByMember(userId).executeAsOneOrNull()?.conversation?.let { conversationId ->
```

Comment:
Question here, is this change related to the scope of the task ? 🤔 Just to understand myself a bit better this how this affects

Reply:
I saw an error that the user was not found in the logs in this function so I secured it

Reply:
Perfect ! 👌